UGW: Add ugw image calls: 'make rootfs firmware fullimage' support.

diff --git a/include/toplevel.mk b/include/toplevel.mk
--- a/include/toplevel.mk
+++ b/include/toplevel.mk
@@ -136,6 +136,9 @@ kernel_menuconfig: prepare_kernel_conf
 kernel_nconfig: prepare_kernel_conf
 	$(_SINGLE)$(NO_TRACE_MAKE) -C target/linux nconfig
 
+rootfs firmware fullimage:
+	@$(_SINGLE)$(SUBMAKE) -C target/linux $@
+
 staging_dir/host/.prereq-build: include/prereq-build.mk
 	mkdir -p tmp
 	rm -f tmp/.host.mk
diff --git a/target/linux/Makefile b/target/linux/Makefile
--- a/target/linux/Makefile
+++ b/target/linux/Makefile
@@ -11,3 +11,6 @@ export TARGET_BUILD=1
 
 prereq clean download prepare compile install menuconfig nconfig oldconfig update refresh: FORCE
 	@+$(NO_TRACE_MAKE) -C $(BOARD) $@
+
+rootfs firmware fullimage:
+	@+$(NO_TRACE_MAKE) -C $(BOARD)/image $@
